### ISSUE TITLE
Movement Controls Disabled By Adding a Custom Key Command

### ISSUE THREAD

**Issue by TruFunkSoulJah** [tsunamayo/Starship-EVO#2821]
### Short Description of the issue:
When changing or customizing the key commands movement controls no longer work.
...

### Steps to reproduce the issue:
Options>Key Mapping>Added "X" as and alt command for "crouch"
Exit> All movement keys are not responding to input.
...

### Environment
- Starship EVO version: 20w34a
- Graphics Card:
AMD Radeon RX 5700 XT 8GB GDDR6
AMD Ryzen 7 3700X
### Additional information:
[output_log.txt](https://github.com/tsunamayo/Starship-EVO/files/5137758/output_log.txt)

_(Screenshots, Videos, Error logs, Related issues)_

Please also include the `output_log.txt` file. You can find your log here: `%UserProfile%\AppData\LocalLow\Moonfire Entertainment\Starship EVO\`

**Comment by tsunamayo** [comment]
@TruFunkSoulJah hi thanks for booking the bug!
Are you sure that this log is the one from the session where you encounter the bug?
Also what is your keyboard region and OS locale? Thanks!

**Comment by TruFunkSoulJah** [comment]
I’m no 100% sure as I’m not very experienced with Windows (former MacOS user). However, it was my first play through since I downloaded the game. Unfortunately (or maybe fortunately) I cannot seem to reproduce the bug. 

I tried landing on another planet and my ship froze and indicated it had been destroyed. Since I was in the middle of space I had to restart with a new character so perhaps it was just an issue with that particular character. 

I’m sorry I can’t be more helpful- I’m not an experienced programmer but love the direction this game is taking and just want to help in any way I can. 

One thing I did notice during the 2nd tutorial is that, as a new player that’s unfamiliar with the controls I accidentally right clicked and deleted the floor of the base. It would be great if all the objects in the tutorial were “un-deletable” to prevent new players from accidentally removing parts of the tutorial. 

I hope that’s helpful. I’m happy to provide more feedback of that nature if it helps. Good luck with this game- it looks great and I’m looking forward to seeing where it goes from here! 
-G-
